packages/metal3/charts/values.yaml

```diff
@@ -31,6 +31,13 @@ global:
     ironicInspectorUsername: ""
     ironicInspectorPassword: ""
 
+  # whether to have additional trusted CA
+  # NOTE: If enabled, a secret with name tls-ca-additional should be deployed
```

Comment:
ok, makes sense as we said, but I'd say to document this example in the official documentation.